### PR TITLE
closes #2 hex logo and various other updates from default admiraltemplate

### LINKED ISSUE #2: Upload hex sticker for admiralpeds
Upload hex sticker for admiralpeds @manciniedoardo do you have a copy?

### PR DESCRIPTION
**Pull request by rossfarrugia**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [ ] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [ ] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [ ] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [ ] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [ ] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [ ] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [ ] Address any updates needed for vignettes and/or templates
- [ ] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [ ] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [ ] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [ ] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [ ] Link the issue in the Development Section on the right hand side.
- [ ] Address all merge conflicts and resolve appropriately
- [ ] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by rossfarrugia** [comment]
@bms63 @manciniedoardo a bunch more minor updates from the admiraltemplate defaults. let me know if you spot any others. FYI i did create https://github.com/pharmaverse/admiralpeds/issues/7 which catches some more needed for later once people dive into this.

File: DESCRIPTION [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 Authors@R: c(
-    person("Open", "Source", email = "admiraltemplate@pharmaverse.com", role = c("aut", "cre")))
+    person("Ross", "Farrugia", email = "ross.farrugia@roche.com", role = c("aut", "cre")))
 Description: A toolbox for programming Clinical Data Standards Interchange Consortium
     (CDISC) compliant Analysis Data Model (ADaM) datasets in R. ADaM datasets are a
     mandatory part of any New Drug or Biologics License Application submitted to the
     United States Food and Drug Administration (FDA). Analysis derivations are
     implemented in accordance with the "Analysis Data Model Implementation Guide"
-    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam/adamig-v1-3-release-package>).
+    (CDISC Analysis Data Model Team, 2021, <https://www.cdisc.org/standards/foundational/adam>).
```
Comment (bms63) [inline DESCRIPTION:12]: Do we still need the year here?  Wondering if we should drop that here and in other admiral packages?

**Review by bms63 — APPROVED** [review]

File: DESCRIPTION (hunk 1 above)
Comment (rossfarrugia) [reply DESCRIPTION:12]: yes could probably be dropped - maybe best to have one person go through all and drop this in one go
